### ISSUE TITLE
alt.Vehicle.gear returns 0 all the time

### ISSUE THREAD

**Issue by MacieGx** [altmp/altv-issues#639]
**Client/server version**
1.0-dev6

**Current behavior**
alt.Vehicle.gear returns 0 all the time
![image](https://user-images.githubusercontent.com/32741786/92449749-ca643f80-f1ba-11ea-9fe8-fb905f6507f4.png)

**Expected behavior**
alt.Vehicle.gear should return the current gear as before.

**Steps to reproduce**
```js
const vehicle = alt.Player.local.vehicle;
editor.log(vehicle.gear);
```

**Comment by martonp96** [comment]
Fixed in dev/1.0-dev13

**Comment by Lhoerion** [comment]
~~Added~~/Fixed/~~Changed~~ in build #[1.0-dev13](https://discord.com/channels/371265202378899476/528297438348574741/756180743759200288)
